src/PhotometryFit.cc

```diff
@@ -214,25 +214,33 @@ void PhotometryFit::offsetParams(Eigen::VectorXd const &delta) {
 
 void PhotometryFit::saveChi2MeasContributions(std::string const &baseName) const {
     std::ofstream ofile(baseName.c_str());
-    /* If we think the some coordinate on the focal plane is relevant in
-       the ntuple, because thmodel relies on it, then we have to add
-       some function to the model that returns this relevant
-       coordinate. */
```

Comment:
While I'll admit this comment isn't enlightening, did you mean to delete it?

Reply:
Yes: it's not relevant to photometry, and must have gotten carried over when I copied the code from astrometry before modifying it.